diagnostics_tools/include/diagnostic_tools/sampled_statistics.h

```diff
@@ -5,31 +5,59 @@
 #ifndef _DIAGNOSTIC_TOOLS__SAMPLED_STATISTICS_H_
 #define _DIAGNOSTIC_TOOLS__SAMPLED_STATISTICS_H_
 
-namespace qna {
-namespace diagnostic_tools {
-
+namespace qna
+{
+namespace diagnostic_tools
+{
 template <typename T>
-class SampledStatistics {
+class SampledStatistics
+{
  public:
-  void reset() {
+
+  SampledStatistics (size_t windows_size){
+    windows_size_ = windows_size;
+  }
+  SampledStatistics (){
+    windows_size_ = 100;
+  }
```

Comment:
@patapufate do we need a default constructor? If it's arbitrary and it has no use for us, I'd suggest we drop it.

Reply:
Fixed.